release-calendar/src/client/models/view-models.ts

```diff
@@ -56,3 +56,13 @@ export class CurrentReleases {
 
   map: Map<Channel, string>;
 }
+
+export class ReleaseDates {
+  constructor(release: Release) {
+    this.dates = release.promotions.map((promotion) => {
+      return {channel: promotion.channel, date: promotion.date};
+    });
+  }
+
+  dates: {channel: Channel; date: Date}[];
+}
```

Comment:
Why do we need a new class when it's just paring down `Promotion`?

Another note, `dates` to be of type `{Channel, Date}[]` is misleading; you would think it would be an array of `Dates`. A more clear class would have Channel and Date members, then you could have a `const releaseDates = ReleaseDate[]`.

But to my point above, this is just a pared down version of `Promotion`